packages/react/src/search/search.react.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Meta, omit, Story, storyOf } from '../../../../docs';
+import { Search, SearchProps } from './search.react';
+
+export default {
+  title: 'React/Search',
+  component: Search,
+  argTypes: {
+    ...omit<HTMLInputElement>('type'),
+    disabled: { table: { type: { summary: 'boolean' } } },
+    placeholder: { control: 'text' },
+  },
+  args: {
+    disabled: false,
+    placeholder: 'Placeholder',
```

Comment:
To align with `Input`:

```suggestion
    placeholder: { control: 'text' },
    maxLength: {
      control: 'number',
      table: { type: { summary: 'number' } },
    },
    disabled: {
      table: { type: { summary: 'boolean' } },
    },
  },
  args: {
    placeholder: 'Placeholder',
    disabled: false,
```

Otherwise `Input` story should be edited.

Reply:
I like the alphabetical table of props
![image](https://user-images.githubusercontent.com/18623773/102613584-a81f7d80-412a-11eb-9886-6aac4af3dfce.png)

But I'll keep them aligned and leave that for a future documentation update.

Reply:
I somehow don't think it will be possible to accomplish this when using the ones coming from type definition + custom table entries. But worth trying.

Reply:
Oh but I'll omit `maxLength` from search, same as `invalid` is omitted, one without the other doesn't make much sense imo.

We should find a good way to document React standard props later.
Right now it's weird, e.g. if we include `maxLength` we should include `minLength`.

Ideally we could just link them and omit from the table.

Reply:
When I added `maxLength` was thinking about `minLength` but it does not do anything. Probably even having `maxLength` is not worth it.

For example, try to sort out  `Button` component table arguments in alphabetical order...

Reply:
🤔 
![image](https://user-images.githubusercontent.com/18623773/102615461-dfdbf480-412d-11eb-86d4-3ac8978e90aa.png)

I can see your point tho, it can happen. In those cases we'd have to manually override them which is annoying

Reply:
So I somehow see limitation here with the plugin - perhaps we could submit a PR and add sorting option? What do you think, your Christmas project? 🎄

Reply:
lol most definitely not, need some rest this holiday season.
But I can take a look when I'm back, sure.